### PR TITLE
Omero

### PR DESCRIPTION
**Pull request by jburel**
This PR adds OMERO as a tool

### CODE REVIEW

File: _data/main_tool_and_resource_list.csv [hunk 1]
```diff
@@ -270,3 +270,4 @@ XNAT,https://www.xnat.org/,"Open source imaging informatics platform. It facilit
 XNAT-PIC Pipelines,https://github.com/szullino/XNAT-PIC,Analysing of single or multiple subjects within the same project in XNAT,,"researcher, data manager, data analysis, XNAT-PIC",
 XNAT-PIC Uploader,https://github.com/szullino/XNAT-PIC,Import tool for multimodal DICOM image datasets to XNAT,,"researcher, data manager, collect, XNAT-PIC",
 Zenodo,https://zenodo.org/,Generalist research data repository built and developed by OpenAIRE and CERN,fairsharing:wy4egf,"data publication, biomol sim",
+Bio-Formats,https://www.openmicroscopy.org/bio-formats/,Bio-Formats is a software tool for reading and writing image data using standardized, open formats,biotools:bio-formats,"ome",
```
Comment (bedroesb) [inline _data/main_tool_and_resource_list.csv:273]:
```suggestion
Bio-Formats,https://www.openmicroscopy.org/bio-formats/,"Bio-Formats is a software tool for reading and writing image data using standardized, open formats",biotools:bio-formats,"ome",
```

Comment (bedroesb) [reply _data/main_tool_and_resource_list.csv]: @jburel Sorry that I took the freedom to fix the page_tag in your markdown page!

**Comment by jburel** [comment]
ok no problem. I can update it if you want. I forgot about the g.doc

File: pages/tool_assembly/omero_assembly.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+tags: [data organisation, storage, data analysis]
+page_tag: ome
+summary: Tool assembly for managing imaging data
+---
+
+## What is OMERO?
+
+OMERO is a software platform for managing, sharing and analysing images data. OMERO supports over proprietary 150 file formats, including all major microscopy formats, medical images, digital pathology images, high content screening, etc. using [Bio-Formats](https://www.openmicroscopy.org/bio-formats/). Bio-Formats is a Java software tool for reading proprietary image data and metadata and writing image data using standardized open formats.
+
+OMERO handles all your images in a secure central repository. Users can **view**, **organize**, **analyze** and **share** data from anywhere via the internet. Users can work with image data and metadata from a Desktop application, from the Web or from 3rd party tools e.g. [Fiji](https://fiji.sc/).
```
Comment (korbinib) [inline pages/tool_assembly/omero_assembly.md:13]:
```suggestion
OMERO handles all your images in a secure central repository. Users can view, [organize](data_organisation), [analyze](analysing) and [share](sharing) data from anywhere via the internet. Users can work with image data and metadata from a Desktop application, from the Web or from 3rd party tools e.g. [Fiji](https://fiji.sc/).
```

**Comment by jburel** [comment]
@bedroesb I have updated the g.doc. The changes should match the change in the csv file

**Comment by bedroesb** [comment]
@jburel thats great! I noticed that the g.doc was up to date, but I forgot to update the "synchronization date" in the sheet. Thanks a lot

File: pages/tool_assembly/omero_assembly.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+{% include image.html file="ome_informatics.png" caption="Schematic overview of the OMERO tool assembly." alt="Schematic overview of the OMERO tool assembly." %}
+
+## Who is OMERO intended for?
+
+OMERO is designed to be an institutional repository. It offers a secure central way for scientists, researchers and data stewards to handle their imaging data. All the image data from a facility can be securely stored and managed, using group permissions and user roles to allow controlled access tailored to your institution. From private repositories for sensitive data to hosting public data for your website and latest publications, the permissions model is designed to meet the range of researchers’ needs. OMERO is tried and tested in hundreds of institutions world-wide, with extensive installation and configuration documentation for system administrators and community support via dedicated mailing lists and forums.
```
Comment (korbinib) [inline pages/tool_assembly/omero_assembly.md:22]:
```suggestion
OMERO is designed to be an institutional repository. It offers a secure central way for scientists, researchers and data stewards to handle their imaging data. All the image data from a facility can be securely stored and managed, using group permissions and user roles to allow controlled access tailored to your institution. From private repositories for [sensitive data](sensitive_data) to hosting public data for your website and latest publications, the permissions model is designed to meet the range of researchers’ needs. OMERO is tried and tested in hundreds of institutions world-wide, with extensive installation and configuration documentation for system administrators and community support via dedicated mailing lists and forums.
```

File: pages/tool_assembly/omero_assembly.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+OMERO is an open-source system designed to be extended and integrated with other tools. The OMERO API allows clients to be written in Java, Python, C++ or MATLAB. The OMERO platform includes a Java client OMERO.insight, a Python-based web client OMERO.web, a Command Line Interface which uses Python, and a Java ImageJ plugin. OMERO also supports a scripting service which allows Python scripts to be run on the server and called from any of the other clients. A vast amount of documentation and code examples for scientists, developers and system administrators are available.
+
+A demo server maintained by the OME team is also available for users wishing to try out, [request an account](https://www.openmicroscopy.org/explore/).
+
+
+{% include image.html file="omero_integration.png" caption="3rd party tools and OMERO" alt="3rd party tools and OMERO." %}
+
+## Which tasks can be solved with OMERO?
+
+OMERO can be used for the day-to-day data management of data. Users can remotely view, annotate, analyze, generate figures ready for publication, etc.
```
Comment (korbinib) [inline pages/tool_assembly/omero_assembly.md:35]:
```suggestion
OMERO can be used for the day-to-day data management of data. Users can remotely view, [handle metadata](metadata_management), [analyze](data_analysis), generate figures ready for publication, etc.
```

File: pages/tool_assembly/omero_assembly.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+---
+title: OMERO
+contributors: [Jean-Marie Burel]
+tags: [data organisation, storage, data analysis]
+page_tag: ome
+summary: Tool assembly for managing imaging data
```
Comment (korbinib) [inline pages/tool_assembly/omero_assembly.md:6]:
```suggestion
summary: Tool assembly for managing microscopy imaging data
```

**Review by korbinib — CHANGES_REQUESTED** [review]
Suggested some linking. As there is also a tool assembly on per-clinical imaging would it help to reference OME(RO) as tool assembly for microscopy imaging?

**Review by bedroesb — APPROVED** [review]
Looks good to me, @korbinib do you also have a look?

File: pages/tool_assembly/omero_assembly.md (hunk 5 above)
Comment (jburel) [reply pages/tool_assembly/omero_assembly.md:6]: OMERO supports many imaging data including jpeg, png. We have also worked with marine biologists taking for example images of seabeds or bird colonies. It is not limited to microscopy data, even if our initial focus was on microscope images.

Note that OMERO supports clinical data too.

Comment (korbinib) [reply pages/tool_assembly/omero_assembly.md:6]: Valid point. Let's keep it like this then.

**Review by korbinib — APPROVED** [review]

**Comment by korbinib** [comment]
@jburel Thank you very much for this!

**Pull request merged**